oauthenticator.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   ]
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rack', '~> 1.4'
+  spec.add_runtime_dependency 'rack', '>= 1.4', '< 2.1'
```

Comment:
hm, rack 2 isn't even out, hard to be sure of compatibility.
also by semantic versioning if this is compatible with 2.0 then it should be compatible < 3. in practice who knows.
this restriction is fine for now, then.